lib/modulesync/renderer.rb

```diff
@@ -10,8 +10,14 @@ def initialize(configs = {})
     end
 
     def self.build(from_erb_template)
```

Comment:
Ah sorry, I see what it's trying to do now - opportunistically use a .erb extension template even when called for a filename _without_ the ERB extension. I assumed that `template_file` always was given the ERB extension, but that's not the case.